l10n_es_aeat_vat_prorrate/tests/test_l10n_es_vat_prorrate.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2016-2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0
+
+from odoo.addons.l10n_es_aeat_mod303.tests.test_l10n_es_aeat_mod303 import (
+    TestL10nEsAeatMod303Base,
+)
+
+
+class TestL10nEsAeatVatProrrateBase(TestL10nEsAeatMod303Base):
+    def setUp(self):
```

Comment:
Change by:
```
@classmethod
    def setUpClass(cls):
```
and all self's by cls, only in this function. Ex:
```
cls.taxes_sale = {
    # tax code: (base, tax_amount)
    "S_IVA21B": (1400, 294),
    "S_IVA21B//neg": (-140, -29.4),
    "S_IVA0": (200, 0),
}
```

Reply:
Done